Setup/UpgradeData.php

```diff
@@ -1,18 +1,26 @@
 <?php
+
 namespace Alma\MonthlyPayments\Setup;
 
 use Alma\MonthlyPayments\Gateway\Config\Config;
+use Alma\MonthlyPayments\Helpers\AlmaClient;
+use Alma\MonthlyPayments\Helpers\ConfigHelper;
+use Alma\MonthlyPayments\Helpers\Exceptions\AlmaClientException;
 use Alma\MonthlyPayments\Helpers\Logger;
 use Alma\MonthlyPayments\Model\Adminhtml\Config\ApiUrl\CustomerCancelUrl;
+use Magento\Framework\App\Config\ScopeConfigInterface;
 use Magento\Framework\App\ResourceConnection;
 use Magento\Framework\Setup\ModuleContextInterface;
 use Magento\Framework\Setup\ModuleDataSetupInterface;
 use Magento\Framework\Setup\UpgradeDataInterface;
 use Magento\Payment\Gateway\Config\Config as ConfigGateway;
 use \Exception;
+use stdClass;
 
 class UpgradeData implements UpgradeDataInterface
 {
+    const PATH_EQUAL = 'path = ?';
```

Comment:
I am not fan of this constant that makes databases request less readable

Reply:
I asked you on slack, it's sonar recommandation. 
???